pkg/ui/statusTable.go

```diff
@@ -44,8 +47,8 @@ func RenderTable(data map[string]string, refreshLastEditedTime bool) error {
 	for p, status := range data {
 		alias, err := db.GetRecord(db.DBName, p, pkg.ProjectAliasBucket)
 		if refreshLastEditedTime {
-			lastEdited = utils.GetLastModifiedTime(db.DBName, p)
-			rec := map[string]string{p: lastEdited}
+			lastEdited, timestamp = utils.GetLastModifiedTime(db.DBName, p)
+			rec := map[string]string{p: fmt.Sprintf("%s-%d", lastEdited, timestamp)}
```

Comment:
Define a function that format a time.Time into this

Reply:
yup , that makes much more sense. so much easier to do than what I did too. 

I'll fix this